cpp/src/slice2cs/Gen.cpp

```diff
@@ -2710,15 +2710,15 @@ Slice::Gen::DispatcherVisitor::writeReturnValueStruct(const OperationPtr& operat
              << (getUnqualified("Ice.Current", ns) + " current")
              << epar;
         _out << sb;
-        _out << nl << "ResponseFrame = new global::Ice.OutgoingResponseFrame(current);";
-        _out << nl << "ResponseFrame.StartReturnValue(";
+        _out << nl << "ResponseFrame = new global::Ice.OutgoingResponseFrame(current.Encoding);";
```

Comment:
See comments on OutgoingResponseFrame. I would also use `Response`/`response` for the OutgoingResponseFrame field/parameter.